frontend/public/components/modals/expand-pvc-modal.tsx

```diff
@@ -70,15 +67,34 @@ const ExpandPVCModal = withHandlePromise((props: ExpandPVCModalProps) => {
           </Trans>
         </p>
         <label className="control-label co-required">{t('modal~Size')}</label>
-        <RequestSizeInput
-          name={t('modal~requestSize')}
-          required
-          onChange={handleRequestSizeInputChange}
-          defaultRequestSizeUnit={requestSizeUnit}
-          defaultRequestSizeValue={requestSizeValue}
-          dropdownUnits={dropdownUnits}
-          testID="pvc-expand-size-input"
-        />
+        <div className="pf-c-input-group">
+          <NumberSpinner
+            className="pf-c-form-control"
+            containerClassName="co-pvc-expand__number-spinner"
+            value={requestSizeValue}
+            onChange={(e: React.FormEvent<HTMLInputElement>) =>
+              setRequestSizeValue(e.currentTarget.value)
+            }
+            changeValueBy={(operation) =>
+              setRequestSizeValue(_.toInteger(requestSizeValue) + operation)
+            }
+            autoFocus
+            required
+            min={_.toInteger(defaultSize?.[0] ?? 0)}
```

Comment:
Is it not possible to have a decimal like `2.5`? This will silently round down.

Reply:
PVC CRD doesn't use decimal values. 1.5 GiB would be expressed in MiB. So the current value would be 1024+512 MiB.

Reply:
I was able to create a PVC with 1.5Gi as the request.

Reply:
Shouldn't that show up as `xyz Mi` in the CR?

Reply:
Example: 
![Screenshot from 2021-04-16 22-09-26](https://user-images.githubusercontent.com/54092533/115056419-74017100-9f00-11eb-8470-6241d11a9dab.png)